packages/db/lib/migrator.mjs

```diff
@@ -45,14 +56,46 @@ async function execute (logger, args, config) {
       'migration-finished',
       (migration) => logger.debug(`completed ${basename(migration.filename)}`)
     )
+    const migrations = await postgrator.getMigrations()
+    const current = await postgrator.getDatabaseVersion()
+
+    if (args.up) {
+      const nextUp = migrations.filter(x => x.version > current)
+        .filter(x => x.action === 'do')
+        .sort((a, b) => a.version - b.version)
+        .map(extractVersionFromMigration)[0]
```

Comment:
@marcopiraccini Why it applies only one migration?

Reply:
Yes, in the `up` case is not so useful, the main use case is for `down`: 
- You do a migration, apply it (you can migrate the last, so simply `db migrate` or `db migrate -u`). 
- You do some test, but you want to do changes without recreating the whole DB, so you simply rollback the last migration quickly  (`db migrate -d`).

Reply:
Maybe it's better to have another key that would be responsible for being able to apply migrations one at a time. And combine it with the -u and -d options if needed. I don't know what it should be called, but I'll use "--one-step" as an example.
```bash
platformatic db migrate
platformatic db migrate -u
platformatic db migrate -d
platformatic db migrate --one-step
platformatic db migrate --one-step -u
platformatic db migrate --one-step -d
```

Reply:
I think we might want to call down "rollback", similarly to https://edgeguides.rubyonrails.org/active_record_migrations.html#rolling-back

Reply:
OK, summarizing, I think we can simplify this:
1) We remove "up" because is basically useless
2) We rename "down" to "rollback" (`-r` / `--rollback`)
Will change this PR according with the above.

Reply:
@mcollina and @ivan-tymoshenko done, can you PTAL?